### PR TITLE
tidy up and bug fixes

### PR DESCRIPTION
**Pull request by tejashari1**

### CODE REVIEW

File: src/Tool.DatabaseScriptRunner.Console/Services/FileTransferService.cs [hunk 1]
```diff
@@ -36,7 +40,7 @@ public async Task<IEnumerable<FileObject>> GetFiles(string folderName)
                 using var reader = new StreamReader(stream);
                 files.Add(new FileObject
                 {
-                    FileName = fileName,
+                    FileName = fileName.Replace(folderName,string.Empty),
```
Comment (dave-wharton) [inline src/Tool.DatabaseScriptRunner.Console/Services/FileTransferService.cs:43]: Might need to think about if there are sub folders. i suspect we will not want to include them, but not sure if there will ever be any!

File: src/Tool.DatabaseScriptRunner.Database/TSql/SqlMigrationRunner.cs [hunk 2]
```diff
@@ -17,8 +16,7 @@ protected override async Task VerifyMigrationsTableExists(string connectionStrin
                     id int,
                     description text,
                     hash text,
-                    appliedAt datetime not null default (getdate()));
-                go";
+                    appliedAt datetime not null default (getdate())); ";
```
Comment (dave-wharton) [inline src/Tool.DatabaseScriptRunner.Database/TSql/SqlMigrationRunner.cs:19]: Interesting that getdate is used rather than utc?

**Review by dave-wharton — APPROVED** [review]
Approved! But a couple of comments there to think about

File: src/Tool.DatabaseScriptRunner.Database/TSql/SqlMigrationRunner.cs (hunk 2 above)
Comment (tejashari1) [reply src/Tool.DatabaseScriptRunner.Database/TSql/SqlMigrationRunner.cs:19]: Will be better to use utcdate now I think about it. Will update it

**Pull request merged**